server.py

```diff
@@ -175,6 +203,32 @@ def save_response(bound_logger, survey_response):
     return invalid
 
 
+def save_feedback_response(bound_logger, survey_feedback_response):
+    bound_logger.info("Saving feedback response")
+    invalid = survey_feedback_response.get("invalid")
```

Comment:
There is potential for silent failure here if any of invalid, survey or period are missing - if the requirement is to store these as `None` in the database in any case, ignore this; if it isn't, it may be better to either wrap these in a try/except block and handle the exception, or let the exception get raised and handle it in the calling code.